src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/dao/GoogleDAO.scala

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.dsde.workbench.gpalloc.dao
+
+import org.broadinstitute.dsde.workbench.gpalloc.db.ActiveOperationRecord
+
+import scala.concurrent.Future
+
+abstract class GoogleDAO {
+  def transferProjectOwnership(project: String, owner: String): Future[String]
```

Comment:
Is there any Google work to do in `transferProjectOwnership`? My understanding was that it'll make calls to internal services. If so, it may be better placed elsewhere.

Reply:
no, the google implementation does nothing. i can probably remove it but something makes me want to keep the placeholder around